md/layout-development.md

```diff
@@ -0,0 +1,78 @@
+# Develop a layout
+
+This pages describe how to create a layout and use it in an application.  
+
+## Definition
+
+A layout is structure of the application and page content. It defines menu positioning, a footer, or any common content that could be displayed on every page of the application.
+Each application can have its own layout, or the same layout can be used consistently to all applications.
+
+There is an example of layout in Bonita Portal, that can be found in the Administrator "Resources" section.  
+It is made of three containers:
+- layout-header  
+            Comes with the living application menu widget to display the navigation of your application.                
+- layout-content  
+            Comes with the living application iframe widget to display pages or forms of your application.  
+- layout-footer  
+            Empty by default
+    
+The layout was created using Bonita UI Designer, so you can export it and customize it with Bonita UI designer. 
+You can add more widgets in the containers layout-header and layout-footer.
+The 'layout.css' file manages the layout behavior, allows to display the content container in full height, 
+and header and footer with fixed size.  
+But it does not allow to add other widgets than the living application iframe into layout-content.
+
+For example, you could:
+
+* Add elements in the layout-header, on top of the menu, like a login box or brand image
+* Add somme content in the layout-footer 
+
+::: warning
```

Comment:
I would remove this note.
It refers to a very old version and another layout than Bonita layout (Default living application layout (deprecated))